Lib/test/test_os.py

```diff
@@ -472,7 +472,7 @@ def test_access_denied(self):
         # force CreateFile to fail with ERROR_ACCESS_DENIED.
         DETACHED_PROCESS = 8
         subprocess.check_call(
-            ['icacls.exe', fname, '/deny', 'Users:(S)'],
+            ['icacls.exe', fname, '/deny', '*S-1-5-32-545:(S)'],
```

Comment:
Yes, using the SID to avoid localized names is correct. Thanks for catching this.